### DISCUSSION TITLE
SDQ Protocol

### DISCUSSION THREAD

**Discussion by oznotes** [micropython/micropython#9430] [Libraries & Drivers]
Is this protocol supported , it seems like similar to 1-wire  but i also can't find any documentation about it .

Best Regards

**Comment by jimmo** [comment]
No.

Your best bet is to start with the sigrok driver and try and adapt that into MicroPython's exisitng `onewire` module. (This is two halves, [high level bits in Python](https://github.com/micropython/micropython-lib/tree/master/micropython/drivers/bus/onewire) and the [low-level in C](https://github.com/micropython/micropython/blob/master/extmod/modonewire.c) ).
